src/extras/render-passes/render-pass-camera-frame.js

```diff
@@ -101,17 +105,25 @@ class RenderPassCameraFrame extends RenderPass {
      */
     layersDirty = false;
 
+    /**
+     * The camera frame that this render pass belongs to.
+     *
+     * @type {CameraFrame}
+     */
+    cameraFrame;
+
     /**
      * @type {RenderTarget|null}
      * @private
      */
     rt = null;
 
-    constructor(app, cameraComponent, options = {}) {
+    constructor(cameraFrame, app, cameraComponent, options = {}) {
```

Comment:
When a constructor takes the app, don't we normally have that argument first in the args list?